src/services/ProductExports.ts

```diff
@@ -0,0 +1,134 @@
+import * as api from '@/routers/productExports';
+import * as db from '@/database/ProductExports';
+import * as productTransactionsDb from '@/database/ProductTransactions';
+import logger from '@/utilities/modules/logger';
+
+/** message when an unhandled error is thrown */
+export const unhandledErrorMsg = 'unhandled error';
+
+/**
+ * Connector for the API and database to get a single product export
+ */
+export async function getProdExportFromDb(uuid: string): Promise<api.ProdExport> {
+  logger.info('getting single transaction', uuid);
+  const dbProdExport: db.ProdExportDb = await db.getProdExport(uuid);
+  const result: api.ProdExport = {
+    uuid: dbProdExport.productexportuuid,
+    transportId: dbProdExport.transportid,
+    datetime: dbProdExport.datetime,
+    productType: dbProdExport.productname,
+    amountOfProduct: dbProdExport.amountofproduct,
+    lastModified: dbProdExport.lastmodified,
+  };
+
+  return result;
+}
+
+
+/**
+ * Connector for the API and database to get all product exports
+ */
+export async function getProdExportsFromDb(): Promise<api.ProdExport[]> {
+  const dbProdExports: db.ProdExportDb[] = await db.getProdExports();
+  const results: api.ProdExport[] = [];
+
+  logger.info('results', results);
+
+  dbProdExports.forEach(function (item: db.ProdExportDb) {
+    const newExport: api.ProdExport = {
+      uuid: item.productexportuuid,
+      transportId: item.transportid,
+      datetime: item.datetime,
+      productType: item.productname,
```

Comment:
if we're already returning the productname, should we also return the product units? it should be in the item already anyways since it was joined with the products table

Reply:
Sure